src/store/sagas/bhr.ts

```diff
@@ -914,3 +923,107 @@ export const deleteAppImageEntityWatcher = createWatcher(
   deleteAppImageEntityWorker,
   DELETE_APP_IMAGE_ENTITY
 );
+
+function* backupAllSignersAndVaultsWorker() {
+  yield put(setBackupAllLoading(true));
+  try {
+    const { primarySeed, id, publicId, subscription, networkType, version }: KeeperApp = yield call(
+      dbManager.getObjectByIndex,
+      RealmSchema.KeeperApp
+    );
+    const encryptionKey = generateEncryptionKey(primarySeed);
+    const walletObject = {};
+    const signersObject = {};
+    const vaultObject = {};
+
+    // update all wallets and signers
+    const wallets: Wallet[] = yield call(dbManager.getCollection, RealmSchema.Wallet);
+    for (const index in wallets) {
+      const wallet = wallets[index];
+      const encrytedWallet = encrypt(encryptionKey, JSON.stringify(wallet));
+      walletObject[wallet.id] = encrytedWallet;
+    }
+    const signers: Signer[] = yield call(dbManager.getCollection, RealmSchema.Signer);
+    for (const index in signers) {
+      const signer = signers[index];
+      const encrytedSigner = encrypt(encryptionKey, JSON.stringify(signer));
+      signersObject[getKeyUID(signer)] = encrytedSigner;
+    }
+    const vaults: Vault[] = yield call(dbManager.getCollection, RealmSchema.Vault);
+    for (const index in vaults) {
+      const vault = vaults[index];
+      const vaultEncrypted = encrypt(encryptionKey, JSON.stringify(vault));
+      const signersData: Array<{
+        signerId: string;
+        xfpHash: string;
+      }> = [];
+      for (const signer of vault.signers) {
+        signersData.push({
+          signerId: getKeyUID(signer),
+          xfpHash: hash256(signer.masterFingerprint),
+        });
+      }
+      vaultObject[vault.id] = {
+        vaultShellId: vault.shellId,
+        vaultId: vault.id,
+        scheme: vault.scheme,
+        signersData,
+        vault: vaultEncrypted,
+      };
+    }
+
+    const nodes: NodeDetail[] = yield call(dbManager.getCollection, RealmSchema.NodeConnect);
+    const nodesToUpdate = [];
+    if (nodes && nodes.length > 0) {
+      for (const index in nodes) {
+        const node = nodes[index];
+        node.isConnected = false;
+        const encrytedNode = encrypt(encryptionKey, JSON.stringify(node));
+        nodesToUpdate.push(encrytedNode);
+      }
+    }
```

Comment:
Does the backupAllSignersAndVaults endpoint in the relay updating the nodes as well?